frontend/assets/stylesheets/components/_bundle-offerings-a.scss

```diff
@@ -0,0 +1,79 @@
+.offering-column,
+.offering-column--left-column{
+    width: 300px;
+    display: inline-block;
+    vertical-align: top;
+    background-color: #d4ebf8;
+    margin-top: 24px;
+}
+
+.offering-column--left-column {
+    margin-right: 20px;
```

Comment:
We should try and use multiples of [`gs-gutter`](https://github.com/guardian/membership-frontend/blob/master/frontend/assets/stylesheets/_mixins.scss#L18) to represent horizontal spacing, to reduce the amount of magic numbers and help preserve a consistent grid pattern